DESCRIPTION

```diff
@@ -61,7 +61,7 @@ Config/testthat/start-first: ause_cassette_re_record
 Encoding: UTF-8
 Language: en-US
 Roxygen: list(markdown = TRUE)
-RoxygenNote: 7.3.2
+RoxygenNote: 7.3.2.9000
```

Comment:
@maelle ROR support requires dev `roxygen2` I assume, but do we know when the next verison of `roxygen2` will hit cran?

Reply:
No I don't know, but I could remove this change from the PR.

Reply:
Yeah, let's go back to the current cran version of `roxygen2` for now, and I've opened an issue to upgrade when `roxygen2` is updated on cran https://github.com/ropensci/vcr/issues/486

Reply:
Done!

Reply:
thanks!